Account/FxAClient10.swift

```diff
@@ -341,6 +378,69 @@ open class FxAClient10 {
 
         return makeRequest(mutableURLRequest, responseHandler: FxADevice.fromJSON)
     }
+    
+    open func oauthAuthorize(withSessionToken sessionToken: NSData, keyPair: RSAKeyPair, certificate: String) -> Deferred<Maybe<FxAOAuthResponse>> {
+        let audience = self.getAudience(forURL: self.oauthURL)
+        let assertion = JSONWebTokenUtils.createAssertionWithPrivateKeyToSign(with: keyPair.privateKey,
+                                                                              certificate: certificate,
+                                                                              audience: audience,
+                                                                              issuer: "127.0.0.1",
+                                                                              issuedAt: Date.now(),
+                                                                              duration: OneYearInMilliseconds)
+        
+        let oauthAuthorizationURL = self.oauthURL.appendingPathComponent("/authorization")
+        var mutableURLRequest = URLRequest(url: oauthAuthorizationURL)
+        mutableURLRequest.httpMethod = HTTPMethod.post.rawValue
+        mutableURLRequest.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        
+        let parameters = [
+            "assertion": assertion,
+            "client_id": AppConstants.FxAiOSClientId,
```

Comment:
This is worked as part of https://github.com/mozilla/fxa-oauth-server/issues/473